package.json

```diff
@@ -16,8 +16,9 @@
   "scripts": {
     "clean": "rimraf lib",
     "build:js": "babel src --out-dir lib --extensions \".ts\" --source-maps inline",
+    "build:language-server": "cd node_modules/@emberwatch/ember-language-server && yarn",
     "lint:js": "eslint . --ext .ts",
-    "build": "npx npm-run-all clean build:js"
+    "prepare": "npx npm-run-all clean build:language-server build:js"
```

Comment:
when is prepare ran?

Reply:
on `npm install` or `yarn install` it will run automatically so the `yarn install --frozen-lockfile` from `.vimrc` that is run by `vim-plug` will trigger it.  same way that coc.nvim builds it's on binaries.

Reply:
nice! this is exciting!